client/web/compose/src/views/Admin/Pages/Builder.vue

```diff
@@ -475,6 +498,18 @@ export default {
         ...this.page.blocks.filter(({ blockID }) => tabbedIDs.has(blockID)),
       ]
     },
+
+    shouldIndicate () {
+      if (!this.editor.block) return
+
+      const { layoutID } = this.$route.query
+      // All blockIDs in other layouts except this one
+      const otherLayoutBlockIDs = this.layouts.filter(({ pageLayoutID }) => pageLayoutID !== layoutID).reduce((acc, { blocks }) => {
```

Comment:
Instead of filtering, just check it during the reduce `if () return`
I would also make this set a computed so you do not have to recompute each time you open a block config